backend/api/Areas/Admin/Controllers/RoleController.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Claims;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Areas.Admin.Models;
+using Pims.Dal.Entities;
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Services.Admin;
+
+namespace Pims.Api.Areas.Admin.Controllers
+{
+    /// <summary>
+    /// RoleController class, provides endpoints for managing roles.
+    /// </summary>
+    // [Authorize (Roles = "administrator")]
```

Comment:
Was this on porpoise?

Reply:
Yes it was, but now that I can get the frontend running I may put it back on.

Reply:
done